initializr-service/src/main/resources/application.yml

```diff
@@ -1292,6 +1300,38 @@ initializr:
           id: remote-shell
           description: CRaSH shell integration
           versionRange: "[1.0.0.RELEASE,1.5.0.RC1)"
+    - name: Spring Cloud GCP
+      bom: spring-cloud-gcp
+      versionRange: 2.0.0.RC1
+      content:
+        - name: GCP Support
+          id: cloud-gcp
+          description: Spring Boot autoconfiguration to use GCP services through Spring Cloud
```

Comment:
We don't use the term "Spring Boot autoconfiguration" anywhere. The whole  site is about getting started with a Spring Boot app. Can we please revisit the 3 descriptions?